### ISSUE TITLE
encoder training error and stop

### ISSUE THREAD

**Issue by fanpengustc** [CorentinJ/Real-Time-Voice-Cloning#119]
when I training encoder without change anything of your code  the program will stop in thousands step，the error is Dataloader stop. when the training start，I discover that  the memory used raise about 100M every step  and  at last  my machine‘s 200G memory will be all used. At first, I think that may be the reason of dataloader of your code. So I run the encoder train with a new dataloader coder write by myself, but that memory raise up still happen and training will stop too. So I think whether the loss of your code cause the memory leak?

**Comment by freecui** [comment]
I have meet the similar issue; The memory used raise when I run the synthesizer_preprocess_embeds.py; I also doubt whether there is the memory leak or not
